src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -173,8 +173,6 @@ The GitHub integration is available for all projects under your Sentry organizat
 
    ![Configure GitHub enterprise](github-e-configure.png)
 
-7. Add any repositories that you want to collect commit data from. _Make sure you have given Sentry access to these repositories in GitHub in the previous steps._
-
```

Comment:
7. Add any repositories that you want to collect commit data from. _Make sure you have given Sentry access to these repositories in GitHub in the previous steps._